scripts/cocoapods_override_sdk.rb

```diff
@@ -9,55 +9,69 @@
 IO.copy_stream(URI.open('https://raw.githubusercontent.com/customerio/customerio-ios/main/scripts/cocoapods_override_sdk.rb'), "/tmp/override_cio_sdk.rb")
 load "/tmp/override_cio_sdk.rb"
 
+# Required by native iOS FCM SDK. 
+# Uncomment the line below if you are installing any of the FCM CIO pods. 
+# use_frameworks! :linkage => :static
+
 target 'SampleApp' do
   # Uncomment only 1 of the lines below to install a version of the iOS SDK 
   pod 'CustomerIO/MessagingPushAPN', '~> 2.1' # install production build 
-  # install_non_production_ios_sdk_local_path('~/code/customerio-ios/')
-  # install_non_production_ios_sdk_git_branch('name-of-ios-sdk-branch')
+  # install_non_production_ios_sdk_local_path('~/code/customerio-ios/', is_app_extension: false, push_service: "apn")
+  # install_non_production_ios_sdk_git_branch('name-of-ios-sdk-branch', is_app_extension: false, push_service: "apn")
```

Comment:
Looks like the `push_service` need to be either `apn` or `fcm` depending on which notification service is being used by the app. Can we please add short comment as a reminder to change this for apps using `fcm`?